### ISSUE TITLE
Windows scaling at 150% makes the chrome extension unresponsive

### ISSUE THREAD

**Issue by paradigmincarnate** [bitwarden/browser#184]
Windows 10
Chrome and bitwarden extension already installed (currently logged out).
Right click desktop
Display Settings
Change the size of text, apps, and other items: 100%
Drag the slider to 150% and apply.
Click the BitWarden icon to show its popup window.
Try to click "login" or "create account" buttons - they're unclickable. They also don't recognise when you hover over them.
Close the BitWarden popup window.

Back in display settings, drag the slider back to 100%.
Don't restart/reload anything.
Click the BitWarden icon to show its popup window.
This time the links/buttons are clickable.

Note: This is a pseudo reproduction.
Actually, I encountered a few different configurations.
* Screen1: 100%. Screen2: 150%. Chrome on Screen2. Bitwarden broken.
* Screen1: 100%. Screen2: 125%. Chrome on Screen2. Bitwarden works.
* Screen1: 100%. Screen2: 150%. Chrome on Screen1. Bitwarden works.
* Screen1: 125%. Screen2: 100%. Chrome on Screen2. Bitwarden broken (differently).
* Screen1: 125%. Screen2: 100%. Chrome on Screen1. Bitwarden works.

**Comment by kspearrin** [comment]
Thanks for the info. We'll have to look into this more. Can you by chance see if another browser like Firefox is affected?

**Comment by KenyaAdams** [comment]
This is an issue that affects all chrome extensions (on Windows).  If you're using Windows 10 you can 'fix' it with this workaround (I believe only works on the creators update or later):
1. Find the chrome.exe file (can right-click a shortcut, click properties, then click Open File Location)
2. Right click chrome.exe and go to properties
3. Click the Compatibility tab
4. Check the 'Override high DPI scaling behavior
5. Set the 'Scaling performed by' drop down to system
You may have to exit Chrome from the taskbar (or reboot) before the changes are applied.

**Comment by kspearrin** [comment]
Thanks @KenyaAdams . Closing since this seems to be a Chrome bug.

**Comment by Techn1x** [comment]
I experienced this behaviour too, which was fixed with the above workaround. It seems to occur for all Chrome browser extensions, so definitely Chrome's issue.

Just to add to the above workaround, when setting the "Scaling performed by" setting, 'System (Enhanced)' was also an option, but that didn't work - definitely need to pick the option simply labelled 'System'. I also had to reboot my system - just exiting Chrome didn't work.
